### DISCUSSION TITLE
needs els to be configured in rescoure.lua or fxmanifest?

### DISCUSSION THREAD

**Discussion by belgamods** [matsn0w/IIS-EmergencyLS-ELS-FiveM#126] [Q&A]
Hi i have a question in the custom cars like police or ambulance your need to set the folder for the miss-els in de xmlfile directory ?

**Comment by matsn0w** [comment]
Hi there, can you please clarify what you mean?

If you mean having to specify each XML file in the config: no, you do not have to do that with MISS ELS!

**Reply by belgamods** [reply]
Tis is my _rescoure.lua i have do that (not so good in english im belgian guy)
```
resource_manifest_version '77731fab-63ca-442c-a67b-abc70f28dfa5'

files {
    'vehicles.meta',
    'carvariations.meta',
    'handling.meta',
	'resources/[standalone]/MISS-ELS/xmlFiles/lpa6.xml',
}

data_file 'HANDLING_FILE' 'handling.meta'
data_file 'VEHICLE_METADATA_FILE' 'vehicles.meta'
data_file 'VEHICLE_VARIATION_FILE' 'carvariations.meta'

is_els 'true'
```

**Reply by matsn0w** [reply]
I'm not sure what you are doing here, but it looks like you are mixing some things up here... Please read the docs carefully. You don't have to edit your vehicle resource(s). Simply creating an XML file and putting it in the `xmlFiles/` directory is enough. MISS ELS will discover it automagically.

**Reply by belgamods** [reply]
Its works now but only the light dont work and al the XML files are in the MISS-ELS/xmlFiles and i dont change the rescoure.lua 
i got a preview here
[https://youtu.be/YzMmNGUdR0Y](url)

**Reply by matsn0w** [reply]
Alright! How does your XML file look like? Make sure you write your own using the [MISS ELS VCF Configurator](https://matsn0w.github.io/MISS-ELS/). All other VCF's are not compatible!
